Activities/Database/ConnectionDialog/ConnectionUIDialog/Workaround/DbWorkarounds.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Data.ConnectionUI;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UiPath.Database.Workaround
+{
+    public static class DbWorkarounds
+    {
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        private static extern IntPtr LoadLibrary(string libname);
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto)]
+        private static extern bool FreeLibrary(IntPtr hModule);
+
+
+        public static void SNILoadWorkaround()
+        {
+#if NETCOREAPP
+            IntPtr Handle = LoadLibrary(Path.GetFullPath(typeof(DataConnectionDialog).Assembly.Location + @"\..\runtimes\win-x64\native\sni.dll"));
+            if (Handle == IntPtr.Zero)
+            {
+                int errorCode = Marshal.GetLastWin32Error();
+                throw new Exception(string.Format("Failed to load library (ErrorCode: {0})", errorCode));
+            }
+#endif
+
+#if NETFRAMEWORK
+            IntPtr Handle = LoadLibrary(Path.GetFullPath(typeof(DataConnectionDialog).Assembly.Location + @"\..\runtimes\win-x86\native\sni.dll"));
```

Comment:
In practice, can we reference the same type here as in the other class (or vice-versa) ?

If so, we can just have one file containing this class and have it added in the other place as a linked file - this results in less duplication.

Also, not entirely sure why we need this class duplicated in the first place.

Reply:
added class as linked file to remove the duplication